src/components/site-header/styles/_Header.scss

```diff
@@ -62,22 +50,14 @@
 }
 
 .site-header-desktop {
-  @extend .border-bottom;
+  border-bottom: 1px solid $gray-200;
```

Comment:
Switches off `@extend` because using it includes an `!important` which makes it difficult to theme/override these styles for theme authors.